vars/cloudFoundryDeploy.groovy

```diff
@@ -117,14 +118,16 @@ def deployCfNative (config) {
         passwordVariable: 'password',
         usernameVariable: 'username'
     )]) {
-        def deployCommand = 'push'
+        def deployCommand = getCfNativeDeployCommand(config)
+
         if (config.deployType == 'blue-green') {
-            deployCommand = 'blue-green-deploy'
             handleLegacyCfManifest(config)
         } else {
             config.smokeTest = ''
         }
 
+        def blueGreenDeployOptions = getCfNativeBlueGreenDeploymentOptions(config)
```

Comment:
Also here I would prefer a name closer to what happens, like `deleteOptionIfRequired()`

Reply:
done